templates/faction/members/index.html

```diff
@@ -23,47 +24,58 @@
+                    6: function (node, table, cellIndex) { return $(node).attr("data-val"); },
+                    7: function (node, table, cellIndex) { return $(node).attr("data-val"); },
+                },
 
-            cssAsc: 'up', cssDesc: 'down', sortList: [[1,0]]
+                cssAsc: 'up', cssDesc: 'down', sortList: [[1, 0]]
+            });
         });
-    });
-</script>
+    </script>
 
-<h2 class="title"><i class="fas fa-users fa-icon-inline"></i>{{faction.name|safe}}'s members<span class="flush-right"><b>Status update:</b> {{faction.memberStatusUpda|ts2date}}&nbsp;&nbsp;<b>List update:</b> {{faction.membersUpda|ts2date}}</span><form class="form-inline"><a id="member-refresh" class="flush-right" href=""><i class="fas fa-sync-alt"></i>&nbsp;Refresh status</a></form></h2>
-<div class="module">
-    <table id="faction-members" class="tablesorter center-large trshadow faction-members">
-        <thead>
-            <tr>
-                <th title="Online status"><i class="fas fa-circle"></i></th>
-                <th>Name</th>
-                <th>Last action</th>
-                <th>Status</th>
-                <th title="Days in Faction">DIF</th>
-                <th>Energy</th>
-                <th>CE Rank &nbsp;&nbsp;<i style="cursor: help;" class="fas fa-question-circle" title="CE ranking is based on NNB, equivalent arsons and the last 250 OC rankings of your members. Global ranking is not part of the API so it can be wrong. The more members you have on YATA the more accurate it is."></i></th>
-                <th>NNB | EA &nbsp;&nbsp;<i style="cursor: help;" class="fas fa-question-circle" title="Natural nerve bar and rough estimation of Equivalent Arsons&#10;&#10;Frauds are assumed arson warehouse&#10;Thefts are assumed steal jackets&#10;Auto theft are assumed steal parked car&#10;2 drug deals remove 1 arson (it corresponds to doing drug crimes at 40NNB)&#10;Computer crimes are assumed steal virus&#10;Murders are assumed Assassinate mob boss&#10;Recent (among the last 250 OC) PH and PA are accounted for&#10;&#10;See TornStats CE predictor for more information"></i> </th>
-                <th>Stats&nbsp;&nbsp;<i style="cursor: help;" class="fas fa-question-circle" title="Only AA members can see these stats"></i></th>
-        </thead>
-        <tbody>
-        {% for member in members %}
-        <tr class="faction-member-refresh" data-val="{{member.tId}}">
-            {% include "faction/members/line.html" %}
-        </tr>
-        {% endfor %}
-        </tbody>
-    </table>
-    <p class="update-info">Last update of the list: {{faction.membersUpda|ts2date}}</p>
-    <p class="update-info">Last update of the members status: {{faction.memberStatusUpda|ts2date}}</p>
+    <h2 class="title"><i class="fas fa-users fa-icon-inline"></i>{{faction.name|safe}}'s members<span
+            class="flush-right"><a href="#"><i class="fas fa-file-export"
```

Comment:
Left this as # for now, on some of the other areas of yata you had some extra functions around this kind of stuff, i cannot see this as needed here howver? @Kivou-2000607

Reply:
mmm I don't understand what you mean. You're talking about the anchor thing?